6. Objects and Methods /attr_reader.rb

```diff
@@ -0,0 +1,25 @@
+class Point
+  #shorter version
+  attr_reader :x, :y  #define accessor methods for our instance variables so users of the Point class can use it
+
+  def initialize(x,y)
+    @x, @y = x,y
+  end
+
+  def to_s #return a string that represents this point
+    "(#@x, #@y)" #interpolate the instance variables into the string
```

Comment:
So if I use string interpolation principles I can get rid of 9,10,11 and just make line 25 into "#{p}" however this doesn't return (1,2) it returns the instance variable. Can I have some guidance?

Reply:
the `to_s` method is a great practice. 

giving an instance of `Point`, when printing it, the `to_s` method is called.

the problem here is the interpolation, it should be:
`"(#{x}, #{y})"`